### PR TITLE
Featured content refactoring

### PR DESCRIPTION
**Pull request by damisul**
Extracted FeaturedContent management from AdminController to separate FeaturedContentsController (and FeaturedContentFeaturesController to manage features).

I've put those two controllers to :admin namespace.

Also I've updated UI to use simple_form for rendering featured_content and featured_content_feature forms

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -400,11 +400,11 @@ body.modal-open{
 .modal{
   padding-right: 0 !important;
 }
-.error, .notice, .success {padding:.8em;margin-bottom:1em;border:2px solid #ddd;}
-.error {background:#FBE3E4;color:#8a1f11;border-color:#FBC2C4;}
+.error, .notice, .success, .alert {padding:.8em;margin-bottom:1em;border:2px solid #ddd;}
```
Comment (damisul) [inline app/assets/stylesheets/application.css:403]: backported those changes from flash.alert from coll_squashed branch

File: app/models/featured_content_feature.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
 class FeaturedContentFeature < ApplicationRecord
   belongs_to :featured_content
+
+  validates :fromdate, :todate, presence: true
```
Comment (damisul) [inline app/models/featured_content_feature.rb:4]: previously we had this check in controller code. I've moved it to model's validation

File: config/locales/active_record.he.yml [hunk 3]
```diff
@@ -7,11 +7,15 @@ he:
...
     period: תקופה
+    title: כותרת
+    fromdate: מתאריך
+    todate: עד תאריך
   activerecord:
     models:
-      person: אדם
-      corporate_body: גוף/מוסד
       authority: מחבר
+      corporate_body: גוף/מוסד
```
Comment (damisul) [inline config/locales/active_record.he.yml:16]: added featured_content and ordered models in alphabetical order

File: config/routes.rb [hunk 4]
```diff
@@ -1,5 +1,12 @@
 include BybeUtils
 Bybeconv::Application.routes.draw do
+  namespace :admin do
```
Comment (damisul) [inline config/routes.rb:3]: This notation is much shorter than previous routing rules set.

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -176,7 +176,7 @@
       <% if current_user %>
         <% if current_user.editor? %>
           <br />
-          <h2><%= link_to t(:back_to_dashboard), url_for(controller: :admin, action: :index) %></h2>
```
Comment (damisul) [inline app/views/layouts/application.html.erb:179]: such notation did not worked for controllers in non-default namespace (e.g. Admin::FeaturedContentsController) so I've switched to url helpers.

File: app/models/featured_content.rb [hunk 6]
```diff
@@ -1,17 +1,19 @@
+# frozen_string_literal: true
+
 class FeaturedContent < ApplicationRecord
-  belongs_to :manifestation
-  belongs_to :authority, inverse_of: :featured_contents
+  belongs_to :manifestation, optional: true
+  belongs_to :authority, inverse_of: :featured_contents, optional: true
   belongs_to :user
+  validates :title, :body, presence: true
```
Comment (damisul) [inline app/models/featured_content.rb:7]: I believe at least those two fields should be mandatory.

File: app/models/featured_content.rb [hunk 7]
```diff
@@ -1,17 +1,11 @@
...
 
   # attr_accessible :title, :body, :external_link
   has_many :featured_content_features, class_name: 'FeaturedContentFeature', dependent: :destroy
-
-  def featured_list
```
Comment (damisul) [inline app/models/featured_content.rb:9]: moved this method to haml view. I believe it is better to have such formatting logic in view

**Review by abartov — APPROVED** [review]

**Pull request merged**